src/content/overview/introduction.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Introduction to Giant Swarm
+description: Giant Swarm is a platform that helps engineering teams to build Cloud Native Developer Platform and operate them without much hassle.
```

Comment:
```suggestion
description: Giant Swarm is a platform that helps engineering teams to build Cloud Native Developer Platforms and operate them without much hassle.
```

Reply:
also here and below you always say "engineers" or "engineerig teams" and call their customers "developers", might be clearer to change that to "platform engineers"